Use the upstream dependency set and local source trees for SimpleX Chat.

 $OpenBSD$

Use the upstream dependency set and local source trees for SimpleX Chat.

--- cabal.project.orig
+++ cabal.project
@@ -1,4 +1,4 @@
-packages: .
+packages: . ../simplexmq ../direct-sqlcipher ../sqlcipher-simple
 -- packages: . ../simplexmq
 -- packages: . ../simplexmq ../direct-sqlcipher ../sqlcipher-simple
 
@@ -11,7 +11,7 @@
 --    coverage: False
 --    library-coverage: False
 
-index-state: 2023-12-12T00:00:00Z
+index-state: 2025-06-01T00:00:00Z
 
 package cryptostore
     flags: +use_crypton
@@ -20,9 +20,6 @@
 
 source-repository-package
     type: git
-    location: https://github.com/simplex-chat/simplexmq.git
-    tag: f03cec7a58ed13a39a52886888c74bcefdb64479
-
 source-repository-package
     type: git
     location: https://github.com/simplex-chat/hs-socks.git
@@ -40,9 +37,6 @@
 
 source-repository-package
     type: git
-    location: https://github.com/simplex-chat/aeson.git
-    tag: aab7b5a14d6c5ea64c64dcaee418de1bb00dcc2b
-
 source-repository-package
     type: git
     location: https://github.com/simplex-chat/haskell-terminal.git
@@ -50,9 +44,6 @@
 
 source-repository-package
     type: git
-    location: https://github.com/simplex-chat/android-support.git
-    tag: 9aa09f148089d6752ce563b14c2df1895718d806
-
 -- TODO this fork is only needed to compile with GHC 8.10.7 - it allows previous base version
 source-repository-package
     type: git
@@ -69,6 +60,3 @@
 -- backported fork due http-5.0
 source-repository-package
     type: git
-    location: https://github.com/simplex-chat/wai.git
-    tag: 2f6e5aa5f05ba9140ac99e195ee647b4f7d926b0
-    subdir: warp
